recipes/eumdac/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "eumdac" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bc320d6ca7ce9f192004de01663ffa91c98ac587638541eee80b5608b733efb8
+
+build:
+  noarch: python
+  entry_points:
+    - eumdac = eumdac.cli:cli
+  number: 0
+  script: "{{ PYTHON }} -m pip install ."
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools
+    - requests >=2.5.0
+    - pyyaml
+
+  run:
+    - python >=3.7
+    - setuptools
+    - requests >=2.5.0
+    - pyyaml
+
+test:
+  imports:
+    - eumdac
+
```

```suggestion
  commands:
    - eumdac -h
  
```
To check that the entry points shenanigans worked, it's probably a good idea to test the cli here.